pom.xml

```diff
@@ -78,7 +89,7 @@
     <parent>
         <artifactId>exasol-version-number-provider-generated-parent</artifactId>
         <groupId>com.exasol</groupId>
-        <version>0.2.3</version>
+        <version>0.2.4</version>
         <relativePath>pk_generated_parent.pom</relativePath>
     </parent>
-</project>
+</project>
```

Comment:
Missing end of line :)
I recommend changing IDE settings to auto-add EOL at EOF